nodejs/eks/cluster.ts

```diff
@@ -1904,6 +2002,10 @@ export const AccessEntryType = {
      * For IAM roles associated with self-managed Windows node groups. Allows the nodes to join the cluster.
      */
     EC2_WINDOWS: "EC2_WINDOWS",
+    /**
+     * For IAM roles associated with EC2 instances that need access policies. Allows the nodes to join the cluster.
+     */
+    EC2: "EC2",
```

Comment:
This was a huge PITA to discover. This new Access Entry type is not documented and required when using a custom node role with EKS Auto Mode.